02-Context-or-Narrative/State-of-PureScript-Documentation-2019.md

```diff
@@ -79,7 +79,7 @@ As a result, others who read the resulting documentation will consider it "poor"
 
 ### Maintaining Documentation's Accuracy
 
-Third, documentation becomes outdated/inaccurate due to changes in the subject of the documentation, especially breaking changes. Two aspects of changes: the 'size' of a change in the subject, and the 'frequency' of these changes.
+Third, documentation becomes outdated/inaccurate when the thing being documented changes in two ways: the 'size' of a change, and the 'frequency' of them.
```

Comment:
Should this be turned into a bullet list to make it more skimmable / make the sentence less long?

Reply:
Yeah, good idea. I'll do that now.

Relatedly, I read some of the following sections which explain these two points, and it seems I was wrong when thinking this section is about "the thing being documented", mainly because the second point, the frequency of changes", seems to be talking about the documentation rather than the thing being documented.

Reply:
Oh. No, I meant the thing being documented when talking about the frequency of changes, not the documentation itself. So, I believe you read that correctly.

As an example, if a library was released as v1.0.0 and then later patched (v1.0.2) and then bumped (v1.1.0) and then patched again (v1.1.1) all within a short time period, that outdates the documentation because it doesn't fully cover the library. However, the 'outdatedness' of the docs is not the same as what occurs in breaking changes.